src/olympia/reviewers/forms.py

```diff
@@ -302,25 +302,20 @@ def create_option(self, *args, **kwargs):
         return option
 
 
-class WithAttachmentMixin:
-    def get_invalid_attachment_file_type_message(self):
-        valid_extensions_string = '(%s)' % ', '.join(VALID_ATTACHMENT_EXTENSIONS)
-        return gettext(
-            'Unsupported file type, please upload an archive '
-            'file {extensions}.'.format(extensions=valid_extensions_string)
-        )
-
-    def clean_attachment_file(self):
-        file = self.cleaned_data.get('attachment_file')
-        if file:
-            if not file.name.endswith(VALID_ATTACHMENT_EXTENSIONS):
-                raise forms.ValidationError(
-                    self.get_invalid_attachment_file_type_message()
+def validate_review_attachment(value):
+    if value:
+        if not value.name.endswith(VALID_ATTACHMENT_EXTENSIONS):
+            valid_extensions_string = '(%s)' % ', '.join(VALID_ATTACHMENT_EXTENSIONS)
+            raise forms.ValidationError(
+                gettext(
+                    'Unsupported file type, please upload an archive '
+                    'file {extensions}.'.format(extensions=valid_extensions_string)
                 )
-        return file
+            )
+    return value
 
 
-class ReviewForm(WithAttachmentMixin, forms.Form):
+class ReviewForm(forms.Form):
```

Comment:
We still want validation that both attachment_file and attachment_input aren't provided simultaneously.  Add that check to `def clean`.